### PR TITLE
style(frontend): change progress color pallet [GIX-2983]

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

New color for step in progress.

### Before

Screenshot 2024-09-19 at 11 49 08

### After

Screenshot 2024-09-19 at 11 41 33

### CODE REVIEW

File: src/frontend/src/lib/styles/global/colors.scss [hunk 1]
```diff
@@ -30,4 +30,5 @@
 
 	// Theme
 	--color-primary: var(--color-blue-ribbon);
+	--color-primary-rgb: 0, 102, 255;
```
Comment (peterpeterparker) [inline src/frontend/src/lib/styles/global/colors.scss:33]: Should we rather define the tailwind color in rgb that way we don't hardcode the equivalent in css?

Comment (peterpeterparker) [reply src/frontend/src/lib/styles/global/colors.scss:33]: Or maybe there is a way to derive the rgb?

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/styles/global/colors.scss:33]: there is a way 

`from var(--color-blue-ribbon) r g b;`

but I have to check if it's compatible all over. Let me check and come back here

File: tailwind.config.js [hunk 2]
```diff
@@ -19,6 +19,7 @@ export default {
 			'light-blue': '#ede7fb',
 			blue: '#3b00b9',
 			'blue-ribbon': '#0066ff',
+			'blue-ribbon-rgb': '0, 102, 255',
```
Comment (peterpeterparker) [inline tailwind.config.js:22]: Just sharing an idea, would following work?

1. Remove `blue-ribbon`
2. Then in CSS:

```
--color-blue-ribbon: rgb(theme(colors.blue-ribbon-rgb));
```

Comment (AntonioVentilii-DFINITY) [reply tailwind.config.js:22]: not only it works, but even better:

```scss
	--color-primary-rgb: var(--color-blue-ribbon-rgb);
	--color-primary: rgb(var(--color-primary-rgb));
```

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Pull request merged**